test_login.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+base_url = 'https://python-qa-training.herokuapp.com/api/auth/login'
+email = 'sievdokymov@lohika.com'
+password = '12345678'
+
+def login(email,password):
+    res = requests.post(base_url, json={"email":email, "password":password})
+    return res
+
+def test_login_valid_credentials():
+    assert login(email,password).status_code == 200
+
+def test_login_invalid_email():
+    email = "wrong@wrong.com"
+    assert login(email,password).status_code == 404
+
+def test_login_invalid_password():
+        password = "wrong"
+        assert login(email, password).status_code == 401 #unathorized
+
+def test_login_empty_values():
+    email = ""
+    password = ""
+    assert login(email, password).status_code == 404
```

Comment:
The empty line at the end of the module is missing.

Reply:
@bochkala fixed in latest version submitted as "homework1-5" pull request - appropriate code style changes were made